utils/fo-installdeps

```diff
@@ -110,13 +110,13 @@ if [ $BUILDTIME ]; then
               lenny|hardy) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.3;;
               squeeze) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               wheezy) apt-get $YesOpt install $YesOpt postgresql-server-dev-9.1;;
-              sid) apt-get  $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               lucid|natty) apt-get $YesOpt install postgresql-server-dev-8.4;;
               isadora) apt-get $YesOpt install postgresql-server-dev-8.4;;
               precise|oneiric|quantal|raring|saucy) apt-get $YesOpt install postgresql-server-dev-9.1;;
               trusty) apt-get $YesOpt install postgresql-server-dev-9.3;;
               jessie) apt-get $YesOpt install postgresql-server-dev-9.4;;
-              xenial) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              xenial|yakkety) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              stretch|buster|sid|zesty|artful) apt-get $YesOpt install postgresql-server-dev-9.6;;
```

Comment:
Using metapackages leads to other problems. We explicitly use specific php and psql versions since we do not want to support all of them.

In my opinion, if one tries to install fossology on a not supported system, one should fail early in the install-deps-step instead of silently later, when some features in the application break.

+1 for the idea with the `./configure` step for verifying dependencies.